eskipfile/remote.go

```diff
@@ -154,6 +158,9 @@ func isFileRemote(remotePath string) bool {
 func (client *remoteEskipFile) DownloadRemoteFile() error {
 	data, err := client.getRemoteData()
 	if err != nil {
+		if errors.Is(err, ErrContentNotChanged) {
```

Comment:
You may also return `changed` boolean flag instead of using sentinel error value

Reply:
Please check the first commit. It started as having a boolean value in `remoteEskipFile` and setting it inside the condition, the Error approach was better. However, if we want to return boolean flag we will need to change function schema